config/locales/en.yml

```diff
@@ -137,12 +142,40 @@ en:
         updated: "Your password was changed successfully. You are now signed in."
         send_instructions: "You will receive an email with instructions about how to confirm your account in a few minutes."
 
+  home_page_alert_html: "Home page alert HTML"
+  hub_signup_case_studies_html: "Hub signup case studies HTML"
```

Comment:
There's a code comment where these are used that says they don't actually work. We could possibly remove them?

Reply:
Yeah, I couldn't get into the details of usage. If we remove the code, we can also remove the translations.